src/App.css

```diff
@@ -27,3 +27,37 @@ button {
   border-radius: 4px;
   background-color: rgb(248 185 42);
 }
+
```

Comment:
just a general nit: usually using rem is better than using px. A good [example](https://dev.to/ochukodotspace/the-battle-of-the-units-px-vs-rem-vs-em-3ka8#:~:text=Now%20PX%20and%20REM%20basically,accessibility%20comes%20into%20the%20picture)